server/handlers/k8sconfig_handler.go

```diff
@@ -289,15 +289,6 @@ func (h *Handler) LoadContextsAndPersist(token string, prov models.Provider) ([]
 	}
 	data, err := utils.ReadFileSource(fmt.Sprintf("file://%s", filepath.Join(h.config.KubeConfigFolder, "config")))
 
-	h.EventsBuffer.Publish(&meshes.EventsResponse{
```

Comment:
This is being sent as event irrespective of err in line290.

Reply:
We need this sort of error reporting. Where was this relocated to?